__tests__/client/service-worker/worker.spec.js

```diff
@@ -40,6 +40,11 @@ beforeEach(() => {
   jest.clearAllMocks();
 });
 
+beforeAll(() => {
+  process.env.ONE_APP_BUILD_VERSION = '5.0.0';
```

Comment:
Out of curiosity, should we just read from the `package.json` here instead of hardcoding it to `5.0.0`?

Reply:
For testing, I think that would work - do you foresee hardcoding causing any issues?

Reply:
Nope, I was just curious